code/game/objects/items/miscellaneous.dm

```diff
@@ -144,6 +144,56 @@
 			augment_list[initial(A.name)] = A
 	return augment_list
 
+/obj/item/choice_beacon/syndicate_trooper
+	name = "Syndicate Trooper Equipment Box"
```

Comment:
```suggestion
	name = "Syndicate Loadout Beacon"
```

Reply:
does it have a sprite to reflect this

the default sprite (as i dont see you marking a change) is a beacon remote